.github/workflows/release.yml

```
@@ -0,0 +1,140 @@
+          # Written with a restrictive umask; the runner is ephemeral but the file is a signing key.
+          chmod 600 onebusaway-android/release.keystore
+          umask 077
+          cat > onebusaway-android/secure.properties <<EOF
+          key.store=release.keystore
+          key.alias=${KEY_ALIAS}
+          key.storepassword=${KEYSTORE_PASSWORD}
+          key.keypassword=${KEY_PASSWORD}
+          EOF
+
+      - name: Verify signing config
+        run: >-
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:65: Double quote to prevent globbing and word splitting

(shellcheck)